static/src/javascripts/projects/commercial/modules/prebid/bidder-config.js

```diff
@@ -60,6 +60,7 @@ export const sonobiBidder: PrebidBidder = {
         ad_unit: config.page.adUnit,
         dom_id: slotId,
         floor: 0.5,
+        appNexusTargeting: config.page.appNexusPageTargeting,
```

Comment:
You might need a stricter dependency on this: 
https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/commercial/modules/build-page-targeting.js#L165